modules/acquia_cms_common/composer.json

```diff
@@ -100,11 +100,6 @@
             },
             "drupal/focal_point": {
                 "3162210 - Preview link accidentally closes the media library": "https://www.drupal.org/files/issues/2020-10-06/3162210-17.patch"
-            },
-            "drupal/checklistapi": {
-                "2311855: Count the default setting on the total percentage counter": "https://www.drupal.org/files/issues/2021-10-26/checklistapi-count-default-values-forprogress-2311855-2.patch",
```

Comment:
first patch is not available in 2.1.x, so we can not remove it.
cc @chandan-singh7929

Reply:
yup , only second patch is merged, hence added the first patch back.